src/util/dom_request.ts

```diff
@@ -1,75 +1,68 @@
 //@ts-nocheck
-(function(global) {
-  var fabric = global.fabric;
-  function addParamToUrl(url, param) {
-    return url + (/\?/.test(url) ? '&' : '?') + param;
-  }
+import { noop } from '../constants';
 
-  function emptyFn() { }
+function addParamToUrl(url, param) {
+  return url + (/\?/.test(url) ? '&' : '?') + param;
```

Comment:
This can/should be replace with `URLSearchParams`
Supported by node as well: https://nodejs.org/api/url.html#class-urlsearchparams

Reply:
And can then be removed since is used only once